### PR TITLE
Add babel runtime and replace es2015 with env

### PR DESCRIPTION
**Pull request by TrySound**
babel runtime allows to reuse the same code of helpers across files and packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,10 +52,12 @@
     "babel-plugin-react-transform": "^3.0.0",
     "babel-plugin-transform-react-constant-elements": "^6.5.0",
```
Comment (STRML) [inline package.json]: These should be removable (pulled in by the react preset)

Comment (TrySound) [reply package.json]: I see it here
https://github.com/STRML/react-grid-layout/blob/master/webpack-examples.config.js#L21-L22

**Comment by TrySound** [comment]
Thoughts?

**Comment by STRML** [comment]
What's the actual reduction in compiled size? I'd rather avoid the large babel-runtime dependency if possible.

**Comment by TrySound** [comment]
react-grid-layout.min.js 77.47K

**Pull request closed without merging**